lib/Sema/TypeCheckDistributed.cpp

```diff
@@ -508,7 +508,11 @@ bool CheckDistributedFunctionRequest::evaluate(
     serializationRequirements = getDistributedSerializationRequirementProtocols(
         getDistributedActorSystemType(actor)->getAnyNominal(),
         C.getProtocol(KnownProtocolKind::DistributedActorSystem));
+  } else if (auto proto = dyn_cast<ProtocolDecl>(DC)) {
+    // FIXME: if it has a n AS defined, we can do checks based on that, otherwise, don't
+    return false;
```

Comment:
This should be expanded, this skips type checking the `distributed func` declared in a protocol right now -- instead, it should check against the declared ActorSystem that it must have